pkg/remote/aws/vpc_supplier_test.go

```diff
@@ -0,0 +1,118 @@
+package aws
+
+import (
+	"context"
+	"testing"
+
+	"github.com/cloudskiff/driftctl/pkg/remote/deserializer"
+	awsdeserializer "github.com/cloudskiff/driftctl/pkg/resource/aws/deserializer"
+
+	"github.com/aws/aws-sdk-go/service/ec2"
+
+	"github.com/aws/aws-sdk-go/aws"
+
+	"github.com/cloudskiff/driftctl/test/goldenfile"
+	mocks2 "github.com/cloudskiff/driftctl/test/mocks"
+	"github.com/stretchr/testify/mock"
+
+	"github.com/cloudskiff/driftctl/mocks"
+
+	"github.com/cloudskiff/driftctl/pkg"
+	"github.com/cloudskiff/driftctl/pkg/resource"
+	"github.com/cloudskiff/driftctl/pkg/terraform"
+	"github.com/cloudskiff/driftctl/test"
+)
+
+func TestVPCSupplier_Resources(t *testing.T) {
+	cases := []struct {
+		test    string
+		dirName string
+		mocks   func(client *mocks.FakeEC2)
+		err     error
+	}{
+		{
+			test:    "no VPC",
+			dirName: "vpc_empty",
+			mocks: func(client *mocks.FakeEC2) {
+				client.On("DescribeVpcsPages",
+					&ec2.DescribeVpcsInput{},
+					mock.MatchedBy(func(callback func(res *ec2.DescribeVpcsOutput, lastPage bool) bool) bool {
+						callback(&ec2.DescribeVpcsOutput{}, true)
+						return true
+					})).Return(nil)
+			},
+			err: nil,
+		},
+		{
+			test:    "mixed default VPC and VPC",
+			dirName: "vpc",
+			mocks: func(client *mocks.FakeEC2) {
+				client.On("DescribeVpcsPages",
+					&ec2.DescribeVpcsInput{},
+					mock.MatchedBy(func(callback func(res *ec2.DescribeVpcsOutput, lastPage bool) bool) bool {
+						callback(&ec2.DescribeVpcsOutput{
+							Vpcs: []*ec2.Vpc{
+								{
+									VpcId:     aws.String("vpc-a8c5d4c1"),
+									IsDefault: aws.Bool(true),
+								},
+								{
+									VpcId: aws.String("vpc-0768e1fd0029e3fc3"),
+								},
+								{
+									VpcId:     aws.String("vpc-020b072316a95b97f"),
+									IsDefault: aws.Bool(false),
+								},
+							},
+						}, true)
```

Comment:
Shouldn't it be `false` here ?

Reply:
Even though this won't change the test itself and its results, we should make it so that it's logic for reading PR purposes.

Reply:
Sure, I've inverted this bool meaning in my mind 👍🏻 nice catch